src/model/CplaceVersion.ts

```diff
@@ -5,7 +5,9 @@ import * as path from 'path';
 export class CplaceVersion {
     private static readonly VERSION_GRADLE = 'version.gradle';
 
-    private static _version: CplaceVersion | undefined = undefined;
+    private static _currentVersion: CplaceVersion | undefined = undefined;
```

Comment:
Why do we have 3 different versions?

Reply:
The `currentVersion` will be removed, but it will remain as optional and we need to support it for backwards compatibility. If `currentVersion` is missing, the other two will be used to calculate the version. One of these might also be missing, so it's easiest to just read all of them.